packages/modal/src/Modal/types.ts

```diff
@@ -18,6 +18,11 @@ export const ModalSize = {
 
 export type ModalSize = typeof ModalSize[keyof typeof ModalSize];
 
+export type ForwardedRef =
+  | React.ForwardedRef<HTMLDivElement | null>
+  | null
+  | undefined;
+
```

Comment:
Interesting type - I'm wondering if we want this to be reusable from somewhere like lib